crates/vault-registry/src/lib.rs

```diff
@@ -248,6 +257,8 @@ decl_module! {
         #[transactional]
         fn withdraw_collateral(origin, amount: DOT<T>) -> DispatchResult {
             let sender = ensure_signed(origin)?;
+            let is_operator = <IsNominationOperator<T>>::get(sender.clone());
+            ensure!(!is_operator, Error::<T>::NominationOperatorCannotWithdrawDirectly);
```

Comment:
How should a nominated vault withdraw collateral? It would be helpful not to have to use a separate call (as it would require more work to configure in clients) but just looking to understand the nomination flow.

Reply:
Neither nominated vaults (operators) nor nominators can withdraw collateral directly through the `vault_registry`. This is to prevent attempts of exposing the other nomination parties to increased liquidation risk (by suddenly lowering collateralization). They need to `request_collateral_withdrawal` and then `execute_collateral_withdrawal` by sending transactions to the `nomination` pallet. `execute` doesn't need request IDs, as it just withdraws every request that matured.